files-override/js/eslint.config.mjs

```diff
@@ -0,0 +1,113 @@
+import globals from 'globals';
+import js from '@eslint/js';
+
+import ember from 'eslint-plugin-ember';
+import emberRecommended from 'eslint-plugin-ember/configs/recommended';
+import gjsRecommended from 'eslint-plugin-ember/configs/recommended-gjs';
+
+import prettier from 'eslint-plugin-prettier/recommended';
+import qunit from 'eslint-plugin-qunit';
+import n from 'eslint-plugin-n';
+
+import emberParser from 'ember-eslint-parser';
+import babelParser from '@babel/eslint-parser';
+
+const esmParserOptions = {
```

Comment:
unlike the upstream blueprint, we have a babel config, so we don't need to specify one here